docs/USAGE-GUIDE.md

```diff
@@ -419,17 +423,22 @@ The "default" and "default_tls13" version is special in that it will be updated
 
 The "rfc9151" security policy is derived from [Commercial National Security Algorithm (CNSA) Suite Profile for TLS and DTLS 1.2 and 1.3](https://datatracker.ietf.org/doc/html/rfc9151).
 
-s2n-tls does not expose an API to control the order of preference for each ciphersuite or protocol version. s2n-tls follows the following order:
+s2n-tls does not expose an API to control the order of preference for each ciphersuite or protocol version. Unless [ChaCha20-Poly1305 boosting](#chacha20-poly1305-boosting) is enabled then s2n-tls follows the following order:
 
-*NOTE*: All ChaCha20-Poly1305 cipher suites will not be available if s2n-tls is not built with an Openssl 1.1.1 libcrypto. The
-underlying encrypt/decrypt functions are not available in older versions.
+*NOTE*: All ChaCha20-Poly1305 cipher suites will not be available if s2n-tls is not built with an Openssl 1.1.1 libcrypto. The underlying encrypt/decrypt functions are not available in older versions.
 
 1. Always prefer the highest protocol version supported
 2. Always use forward secrecy where possible. Prefer ECDHE over DHE.
 3. Prefer encryption ciphers in the following order: AES128, AES256, ChaCha20, 3DES, RC4.
 4. Prefer record authentication modes in the following order: GCM, Poly1305, SHA256, SHA1, MD5.
 
-The following chart maps the security policy version to the signature scheme supported:
+#### ChaCha20-Poly1305 Boosting
+
+The prioritization of ChaCha20 ciphersuites is known as ChaCha20 boosting. An s2n-tls server will prioritize its ChaCha20 ciphersuites over its non-ChaCha20 ciphersuites if the server configures a security policy with ChaCha20 boosting enabled AND the client sends a ChaCha20 ciphersuite as its most preferred ciphersuite. 
```

Comment:
As a customer, this doesn't tell me whether or not I want ChaCha20 boosting. You need to explain what this feature is and why I would want it.

Talk about the use case where some clients, particularly mobile clients, don't have AES optimized and so will prefer Chacha20, and indicate that support by listing it as their first choice cipher suite. If chacha20 boosting is enabled, we honor that preference for the sake of performance, rather than sticking to server order.